From: Dan Minor <dminor@mozilla.com>
Date: Tue, 11 Feb 2020 17:07:00 +0000
Subject: Bug 1578073 - Remove native calls in Histogram.java; r=ng

Getting the JNI calls here to work requires a good amount of webrtc.org
machinery. It might be worth setting that up the next time we do an upstream
merge, but for now, it is a lot simpler to just remove the affected code,
given that we are not interested in collecting this data anyway.

Differential Revision: https://phabricator.services.mozilla.com/D61860
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/7a9b07dec9f9d435416b06829fa5063aca3a476e
---
 sdk/android/src/java/org/webrtc/Histogram.java | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/sdk/android/src/java/org/webrtc/Histogram.java b/sdk/android/src/java/org/webrtc/Histogram.java
index 097e740820..90eda633b8 100644
--- a/sdk/android/src/java/org/webrtc/Histogram.java
+++ b/sdk/android/src/java/org/webrtc/Histogram.java
@@ -29,23 +29,13 @@ class Histogram {
   }
 
   static public Histogram createCounts(String name, int min, int max, int bucketCount) {
-    return new Histogram(HistogramJni.get().createCounts(name, min, max, bucketCount));
+    return new Histogram(0);
   }
 
   static public Histogram createEnumeration(String name, int max) {
-    return new Histogram(HistogramJni.get().createEnumeration(name, max));
+    return new Histogram(0);
   }
 
   public void addSample(int sample) {
-    HistogramJni.get().addSample(handle, sample);
-  }
-
-  @NativeMethods
-  interface Natives {
-    long createCounts(String name, int min, int max, int bucketCount);
-
-    long createEnumeration(String name, int max);
-
-    void addSample(long handle, int sample);
   }
 }
